tests/src/container-smoke.spec.ts

```diff
@@ -158,4 +167,35 @@ describe('Verification of container creation workflow', async () => {
     playExpect(await containersPage.containerExists(containerToRun)).toBeFalsy();
     await pdRunner.screenshot('containers-container-deleted.png');
   });
+
+  test('Prune containers', async () => {
+    const navigationBar = new NavigationBar(page);
+
+    for (const container of containerList) {
+      const images = await navigationBar.openImages();
+      const imageDetails = await images.openImageDetails(imageToPull);
+      const runImage = await imageDetails.openRunImage();
+      const containersPage = await runImage.startContainer(container);
+      await playExpect(containersPage.heading).toBeVisible();
+      await playExpect
+        .poll(async () => await containersPage.containerExists(container), { timeout: 15000 })
+        .toBeTruthy();
+    }
+
+    for (const container of containerList) {
+      let containersPage = new ContainersPage(page);
+      const containersDetails = await containersPage.openContainersDetails(container);
+      await playExpect(containersDetails.heading).toBeVisible();
+      await playExpect(containersDetails.heading).toContainText(container);
+      playExpect(await containersDetails.getState()).toContain(ContainerState.Running);
+      await containersDetails.stopContainer();
+      await playExpect(await containersDetails.getStateLocator()).toHaveText(ContainerState.Exited, { timeout: 20000 });
+      containersPage = await navigationBar.openContainers();
+      await playExpect(containersPage.heading).toBeVisible();
+      await containersPage.pruneContainers();
```

Comment:
no, because this way we're also checking that containers that are still in use don't get pruned. The test stops the containers one by one and calls each time prune command and checks that only the correct container is removed.